perfscale_regression_ci/scripts/scalability/utils/cleanup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import time
+import sys
+import ocp_utils 
+
+def delete_all_namespaces(job):
+    # make sure all namespaces are gone
+    print("job type " + str(job))
+    if job.lower() == "pod-density":
+        job = "node-density"
+    ocp_utils.run("oc delete ns --wait=false -l kube-burner-job=" + job)
```

Comment:
This python file did cleanup for kube-burner specific resources. I suggest to change the name cleanup.py to a more specific one, like cleanup_kubeburner.py. 
A 'nice to have' suggestion, I think this can be moved to the [common place](https://github.com/openshift/svt/tree/master/perfscale_regression_ci/scripts)  so test cases under other sub-folders (network/performance) can also make use of it.